cluster/pulumi/deployment/src/stacks/splice.ts

```diff
@@ -28,9 +28,9 @@ export function getSpliceStacksFromMainReference(): StackFromRef[] {
   if (config.envFlag('SPLICE_DEPLOY_MULTI_VALIDATOR', false)) {
     ret.push({ project: 'multi-validator', stack: `multi-validator.${CLUSTER_BASENAME}` });
   }
-  if (DeployValidatorRunbook) {
-    ret.push({ project: 'validator-runbook', stack: `validator-runbook.${CLUSTER_BASENAME}` });
-  }
+  deployedValidators.forEach(validator => {
+    ret.push({ project: 'validator-runbook', stack: `${validator}.${CLUSTER_BASENAME}` });
```

Comment:
stack should be the same as here https://github.com/hyperledger-labs/splice/blob/main/cluster/pulumi/deployment/src/stacks/validators.ts#L22 :cry:

Reply:
fixed, we should just change `deployeValidators` to be a list of objects with all fields we need instead of inlining special casing based on the name all over the place. But not for this pr

Reply:
yeah, it started with a single case and then it was needed in another place and now it's the magic third time we need it